src-docs/src/views/accordion/accordion.js

```diff
@@ -1,13 +1,18 @@
 import React from 'react';
 
 import { EuiAccordion, EuiPanel } from '../../../../src/components';
+import { useGeneratedHtmlId } from '../../../../src/services';
 
-export default () => (
-  <div>
-    <EuiAccordion id="accordion1" buttonContent="Click me to toggle">
-      <EuiPanel color="subdued">
-        Any content inside of <strong>EuiAccordion</strong> will appear here.
-      </EuiPanel>
-    </EuiAccordion>
-  </div>
-);
+export default () => {
+  const simpleAccordionId = useGeneratedHtmlId({ prefix: 'accordion' });
+
+  return (
+    <div>
+      <EuiAccordion id={simpleAccordionId} buttonContent="Click me to toggle">
```

Comment:
Just a quick thought here and I think this might have stemmed from my bad communication of an option I presented. But naming the constant something specific is ok (it might be more than is needed) but I what I was hoping is that the **rendered** ID is what contained that uniqueness.


What this would help is in scenarios where someone is inspecting the rendered DOM and then trying to find the matching component. Currently all these accordion examples instances will still just be `accordion_[hash]` vs something like `simpleAccordion__[hash]`. 

This is just something I struggle with a lot on the consumer side is trying to find where a specific UI was built in the code and using this `prefix` helps sooo much if it is unique.

Ex:
```suggestion
  const simpleAccordionId = useGeneratedHtmlId({ prefix: 'simpleAccordion' });

  return (
    <div>
      <EuiAccordion id={simpleAccordionId} buttonContent="Click me to toggle">
```

I know this has been **a lot** of work so I don't want to necessarily add on to it. But I think the extra step to copy that constant name into the prefix would be a **huge** improvement.

Reply:
Ok! I can make that change for the IDs. I completely agree that if you spend a lot of time in the console, this could be helpful. I formatted the IDs this way based on the feedback I got in my initial PR [5114](https://github.com/elastic/eui/pull/5114). 

If we're if we're the `prefix` to be the example name (and not the component), can you confirm that the format for the IDs should be something like:
`const exampleNameId = useGeneratedHtml({ prefix:'exampleName' });`

If there are multiples (like with keypad buttons), it should look something like:
`const exampleNameId__1 = useGeneratedHtml({ prefix:'exampleName' , suffix: 'first'});`
`const exampleNameId__2 = useGeneratedHtml({ prefix:'exampleName' , suffix: 'second'});`
`const exampleNameId__3 = useGeneratedHtml({ prefix:'exampleName' , suffix: 'third'});`

Luckily you caught me before I created the other PR, so I'll need to go through and update these file.

Reply:
Yeah exactly. That would be ideal.